docs/client_reference.rst

```diff
@@ -860,8 +881,11 @@ TCPConnector
 
       .. versionadded:: 0.21
 
+   :param bool force_close: close underlying sockets after
+                            connection releasing (optional).
+
    :param tuple enable_cleanup_closed: Some ssl servers do not properly complete
-      ssl shutdown process, in that case asyncio leaks ssl connections.
+      SSL shutdown process, in that case asyncio leaks ssl connections.
```

Comment:
There were two ssl, one left (:

Reply:
Thank you very much for such meticulous review!
Fixed.